eng/Version.Details.xml

```diff
@@ -331,18 +321,41 @@
     <Dependency Name="Microsoft.Web.Xdt" Version="9.0.0-preview.24067.1">
       <Uri>https://github.com/dotnet/xdt</Uri>
       <Sha>d71290db981c297b17054b64b2bc7c707a547545</Sha>
-      <SourceBuild RepoName="xdt" ManagedOnly="true" />
     </Dependency>
+    <!-- Intermediate is necessary for source build. -->
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.roslyn" Version="4.8.0-3.23518.7">
+      <Uri>https://github.com/dotnet/roslyn</Uri>
+      <Sha>1aa759af23d2a29043ea44fcef5bd6823dafa5d0</Sha>
+      <SourceBuild RepoName="roslyn" ManagedOnly="true" />
+    </Dependency>
+    <!-- Intermediate is necessary for source build. -->
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.runtime.linux-x64" Version="9.0.0-preview.2.24080.11">
```

Comment:
I personally find it more useful to group the dependencies by Repo.  Do others have an opinion?

Reply:
I agree with this. I've re-arranged the intermediates in https://github.com/dotnet/aspnetcore/pull/53749/commits/8de991405ba4881a2a59a581b5b0bc4af48adba1